Cannot resolve method 'build()' when i try use project clone in Git

Hi i try compile the project in git but when i try create new ACRAConfiguration with ConfigurationBuilder  i have this error

![schermata 2017-04-03 alle 13 39 33](https://cloud.githubusercontent.com/assets/26813554/24607677/395eb200-1873-11e7-844c-54b724148afb.png)

Error:(26, 21) error: cannot find symbol method build()

Just a guess here: go into ConfigurationBuilder and remove the @hide
annotation from build()

Am 03.04.2017 1:41 nachm. schrieb "VincenzoWiman" <notifications@github.com
>:

> [image: schermata 2017-04-03 alle 13 39 33]
> <https://cloud.githubusercontent.com/assets/26813554/24607677/395eb200-1873-11e7-844c-54b724148afb.png>
>
> Error:(26, 21) error: cannot find symbol method build()
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/ACRA/acra/issues/568#issuecomment-291119293>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AJxxpOvs9XdF0AKePA89ZvUgK-DD3HM_ks5rsNr_gaJpZM4MxT-J>
> .
>

--- a/acra/src/main/java/org/acra/annotation/NoPropagation.java
+++ b/acra/src/main/java/org/acra/annotation/NoPropagation.java
@@ -30,5 +30,5 @@ import java.lang.annotation.Target;
 
 @Retention(RetentionPolicy.SOURCE)
 @Target(ElementType.METHOD)
-public @interface Hide {
+public @interface NoPropagation {
 }

--- a/acra/src/main/java/org/acra/config/ConfigurationBuilder.java
+++ b/acra/src/main/java/org/acra/config/ConfigurationBuilder.java
@@ -20,7 +20,7 @@ import android.support.annotation.NonNull;
 
 import org.acra.ACRA;
 import org.acra.ReportField;
-import org.acra.annotation.Hide;
+import org.acra.annotation.NoPropagation;
 import org.acra.dialog.CrashReportDialog;
 import org.acra.sender.HttpSender;
 
@@ -71,7 +71,7 @@ public final class ConfigurationBuilder extends BaseConfigurationBuilder{
      * @return new ACRAConfiguration containing all the properties configured on this builder.
      * @throws ACRAConfigurationException if the required values for the configured notification mode have not been provided.
      */
-    @Hide
+    @NoPropagation
     @NonNull
     public ACRAConfiguration build() throws ACRAConfigurationException {
 
@@ -154,7 +154,7 @@ public final class ConfigurationBuilder extends BaseConfigurationBuilder{
         return httpHeaders;
     }
 
-    @Hide
+    @NoPropagation
     @NonNull
     @Override
     ReportField[] customReportContent() {

--- a/annotationprocessor/src/main/java/org/acra/ModelUtils.java
+++ b/annotationprocessor/src/main/java/org/acra/ModelUtils.java
@@ -50,7 +50,7 @@ class ModelUtils {
     private static final String IMMUTABLE_MAP = "org.acra.collections.ImmutableMap";
     private static final String IMMUTABLE_LIST = "org.acra.collections.ImmutableList";
     private static final String IMMUTABLE_SET = "org.acra.collections.ImmutableSet";
-    private static final String ANNOTATION_HIDE = "org.acra.annotation.Hide";
+    private static final String ANNOTATION_HIDE = "org.acra.annotation.NoPropagation";
 
     private final Types typeUtils;
     private final TypeMirror map;

